tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -0,0 +1,265 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <array>
+#include <catch.hpp>
+#include <random>
+
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/Wedge3D.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Sphere",
+                  "[Domain][Unit]") {
+  const std::array<double, 3> lower_corner{{1.0, 1.0, -1.0}};
+  const std::array<double, 3> upper_corner{{1.0, 1.0, 1.0}};
+  const CoordinateMaps::Wedge3D map(sqrt(3.0), 2.0 * sqrt(3.0),
+                                    Direction<3>::upper_zeta(), 0);
+
+  CHECK(map(lower_corner)[0] == approx(1.0));
+  CHECK(map(lower_corner)[1] == approx(1.0));
+  CHECK(map(lower_corner)[2] == approx(1.0));
+  CHECK(map(upper_corner)[0] == approx(2.0));
+  CHECK(map(upper_corner)[1] == approx(2.0));
+  CHECK(map(upper_corner)[2] == approx(2.0));
+
+  const std::array<double, 3> test_point1{{-1.0, 0.25, 0.0}};
+  const std::array<double, 3> test_point2{{1.0, 1.0, -0.5}};
+  const std::array<double, 3> test_point3{{0.7, -0.2, 0.4}};
+
+  test_forward_jacobian(map, test_point1);
+  test_forward_jacobian(map, test_point2);
+  test_forward_jacobian(map, test_point3);
+
+  test_inverse_jacobian(map, test_point1);
+  test_inverse_jacobian(map, test_point2);
+  test_inverse_jacobian(map, test_point3);
+
+  // Test that the implementation works when used with a CoordinateMap
```

Comment:
This set of checks should probably be done for every map.  Let's split it out into a helper function.